components/automate-gateway/api/iam/v2/common/policy.proto

```diff
@@ -24,42 +31,54 @@ message Statement {
         ALLOW = 0;
         DENY = 1;
     }
+    // Whether the statement allows or denies.
+    // One of `ALLOW`, or `DENY`, respectively.
     Effect effect = 1;
 
-    // inline definitions
+    // DEPRECATED: Actions defined inline. Use role instead.
```

Comment:
Sorry--when did inline actions become deprecated?

Reply:
if a policy has a role it should not have actions, idk if that needs to be communicated here or not. inline actions are allowed if not using a role, but i would prefer not to encourage there use whenever possible.